manifests/add-cluster/kustomize/secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: csdp-add-cluster-secret
+type: Opaque
+data:
+  csdpToken: <csdpToken>
```

Comment:
Maybe we need to think about some way to clean those after the job is done. But not crucial for now.

Reply:
the script is deleting the secret after it successfully finishes.
if it fails - it doesn't

Reply:
gr8